benchmarking/collect_lp_benchmark.sh

```diff
@@ -24,6 +24,19 @@
 # * Benchmark of Barrier LP solvers: http://plato.asu.edu/ftp/lpbar.html
 # * Large Network-LP Benchmark (commercial vs free):
 #     http://plato.asu.edu/ftp/network.html
+#
+# The instances included in Mittelmann's benchmarks is updated periodically to
```

Comment:
Nit (here and below): "The instances included [...] are" or "The set of instances included [...] is"

Reply:
Done (chose "set of instances", and tweaked some nearby wording.